packages/zoe/src/zoeService/startInstance.js

```diff
@@ -50,6 +50,12 @@ export const makeStartInstance = (
     seatHandleToZoeSeatAdmin,
   );
 
+  const getFreshZcfBundleCap = async () => {
+    const settledBundleCap = await getZcfBundleCapP();
+    settledBundleCap !== undefined || Fail`the ZCF bundle cap was broken`;
```

Comment:
When might `settledBundleCap` be `undefined`? What I'm wondering is if "broken" is the right diagnosis to report.

Reply:
When a fresh ZcfBundleCap is gotten, what if anything updates `state.zcfBundleCap`? Does `getZcfBundleCapP` do that? Where is `getZcfBundleCapP` defined anyway?

Reply:
`zcfBundleCap` is defined in zoe.js. It starts out undefined, and is set when `setVatAdminSvc()` is called. Zoe ensures that it's set early, but I think the type system can't tell, and is happier when we check and throw.  "unset" might be more apt, but if it happens, I want whoever sees it to realize it's a big deal.

> When a fresh ZcfBundleCap is gotten, what if anything updates state.zcfBundleCap? 

Ooh! good catch! There's no `state.zcfBundleCap`, it's held in baggage. (We can't change what's in state in a new incarnation.)  A fresh one would be set in a call to `updateZcfBundleId`, which does not update zoeBaggage.  Created an issue: #8805.

> Where is getZcfBundleCapP defined anyway?

In the call to [`makeStartInstance`](https://github.com/Agoric/agoric-sdk/blob/09e90eeb90153b39018afe6309ebdcafa02b07b2/packages/zoe/src/zoeService/zoe.js#L152) it's set to `() => zcfBundleCap`. (i.e. just dynamically access the local version held by Zoe.)